awx_collection/plugins/modules/tower_job_wait.py

```diff
@@ -48,6 +48,11 @@
       description:
         - Maximum time in seconds to wait for a job to finish.
       type: int
+    job_type:
+      description:
+        - Job type to wait for
+      choices: ['project_updates', 'jobs', 'inventory_updates', 'workflow_jobs']
+      default: 'jobs'
```

Comment:
`type: str` needs to be specified here.

Reply:
I addressed this one as well. I will pull in devel and add testing next.

Reply:
Awesome, thank you @donald-picard-kr !

Reply:
> @donald-picard-kr thank you for this PR! Could you please rebase against the `devel` branch and also add a task or two into the [integration test playbook](https://github.com/ansible/awx/blob/devel/awx_collection/tests/integration/targets/tower_job_wait/tasks/main.yml) to check that the new `job_type` parameter is valid? I also added a comment above about `type: str` needing to be specified in the module options.

@beeankha I think I have addressed all of your comments.